src/main/java/decodes/hdb/HDBRatingTable.java

```diff
@@ -122,9 +124,11 @@ public void findRating(Date valueDate)
 		else try 
```

Comment:
I don't think you wrote this bit but this line strongly implies a syntax structure that doesn't exist in java. Just wanted to document it for later review.